tdvt/tdvt/setup_env.py

```diff
@@ -222,7 +222,7 @@ def mangle_tds(file_path, connection_password_name):
     try:
         r1 = re.compile('(^\s*<named-connection .*? name=\').*?(\'>)')
         r2 = re.compile('(^\s*<.*relation connection=\').*?(\' .*>)')
-        r3 = re.compile('(^\s*<connection .*?)(\s*/>)')
+        r3 = re.compile('(^\s*<connection .*?)''(^\s*<tdvtconnection .*?)(\s*/>)')
```

Comment:
`tdvtconnection` is an attribute within the `connection` element. The `<` doesn't look right to me. Were you able to test this somehow?